<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="matrixTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>a</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>b</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>c</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>9</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="operationComboBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>290</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Gauss Elimination</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gauss-Jordan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gauss-Seidel </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jacobi-Iteration</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Doolittle Decomposition</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Crout Decomposition</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cholesky Decomposition</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="solveButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Solve</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="vectorTable">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>x1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>x2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>x3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="solutionLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Solution:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="solutionMatrix">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>x1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>x2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>x3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
